cpp/include/cuvs/neighbors/ivf_pq.h

```diff
@@ -459,3 +459,6 @@ cuvsError_t cuvsIvfPqExtend(cuvsResources_t res,
 #ifdef __cplusplus
 }
 #endif
+
+// Include Multi-GPU support
+#include <cuvs/neighbors/mg_ivf_pq.h>
```

Comment:
Can we please just put the functions right in this file so we don't need to have separate headers for these? Ideally, we'd try to minimize the number of headers because these are very very lightweight (just prototype definitions). Especially if we are always going to include these, I'd prefer just to put them right in `ivf_pq.h`, for example.